### PR TITLE
update mac installation instructions

### PR DESCRIPTION
**Pull request by indirectlylit**
The mac installation instructions are still a bit confusing and over-complicated. The nested list is also strangely formatted.

Trying to simplify things a bit here:

| before | after |
|--|--|
| image |image|

This also provides a direct link to the DMG due to https://github.com/learningequality/learningequality.org/pull/486

### CODE REVIEW

**Comment by radinamatic** [comment]
Can we please first get the #157 merged?

**Comment by benjaoming** [comment]
@radinamatic not sure about the timing of this, but we can link up if you would like some tricks on how to quickly merge @indirectlylit 's PR so it doesn't interfere with other work you're doing?

**Comment by radinamatic** [comment]
I don't plan to add anything more in #157 (no tricks are needed), if there are no remaining blockers, let's merge it.

**Comment by benjaoming** [comment]
Ah okay, I get you! It's merged!

**Review by radinamatic — APPROVED** [review]
Great editing work, looks much better, thank you!

File: docs/install/macos.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Install
 
 .. note:: The Kolibri Mac app is not signed, which means that you must tell MacOS to allow it to run. Read `more information about opening unsigned apps <https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac>`_ from Apple.
 
-#. Download the `.dmg installer <https://learningequality.org/download/>`_ for Kolibri **version 0.13**.
+#. Download the `.dmg installer <https://learningequality.org/r/kolibri-mac-latest>`_ for Kolibri **version 0.13**.
```
Comment (indirectlylit) [inline docs/install/macos.rst]: updating to a direct link due to  https://github.com/learningequality/learningequality.org/pull/486

**Pull request merged**